javascript-forms/form-values.js

```diff
@@ -0,0 +1,13 @@
+var $form = document.querySelector('#contact-form');
+var $name = document.querySelector('#user-name');
+var $email = document.querySelector('#user-email');
+var $message = document.querySelector('#user-message');
+var $collection = {};
```

Comment:
You should be making this in the function, since you don't want this variable to continue to exist after the function is done running 👍